docs/guides/typescript.md

```diff
@@ -190,11 +190,11 @@ const useBearStore = create<BearState>()(
 )
 ```
 
-Also it's recommended to use `devtools` middleware as last as possible, in particular after `immer` middleware, ie it should be `immer(devtools(...))` and not `devtools(immer(...))`. The reason being that `devtools` mutates the `setState` and adds a type parameter on it, which could get lost if other middlewares (like `immer`) mutate `setState` before `devtools`.
+Also, we recommend using `devtools` middleware as an argument, rather than passing arguments to it. For example, when you use it with `immer` as a middleware, it should be `immer(devtools(...))` and not `devtools(immer(...))`. This is because`devtools` mutates the `setState` and adds a type parameter on it, which could get lost if other middlewares (like `immer`) mutates `setState` before `devtools`.
```

Comment:
> we recommend using `devtools` middleware as an argument, rather than passing arguments to it

This sounds a bit wrong, even if you use `devtools` at the end you're still passing an argument to it namely your state creator.

I think the original wording is better, I'm fine if someone can come up with a better one but the one is PR isn't really making the cut imo.

Also "mutate" was correct because it's "middlewares".

```suggestion
Also, we recommend using `devtools` middleware as last as possible. For example, when you use it with `immer`, it should be `immer(devtools(...))` and not `devtools(immer(...))`. This is because `devtools` mutates `setState` and adds a type parameter on it, which could get lost if other middlewares (like `immer`) also mutate `setState` before `devtools`.
```